compose.yaml

```diff
@@ -18,6 +18,21 @@ services:
       context: ./frontend/
     ports:
       - 5001:8080
+  mongo:
+    build:
+      context: ./database/
+    ports:
+      - 5002:27017
```

Comment:
```suggestion
      - "${MONGO_PORT}:27017"
```

ports should generally not be hardcoded

Reply:
also relevant for the other services, frontend & backend

Reply:
Done